launcher/src/test/java/org/apache/brooklyn/launcher/blueprints/Windows7zipBlueprintLiveTest.java

```diff
@@ -81,7 +81,7 @@ public void test7zip() throws Exception {
         
         Predicate<Application> asserter = new Predicate<Application>() {
             @Override public boolean apply(Application app) {
-                VanillaWindowsProcess entity = Iterables.getOnlyElement(Entities.descendants(app, VanillaWindowsProcess.class));
+                VanillaWindowsProcess entity = Iterables.getOnlyElement(Entities.descendantsAndSelf(app, VanillaWindowsProcess.class));
```

Comment:
The author of this test clearly only wanted self here - is it worth adding a self method to `Entities`? (or if this comes up a lot a `rootElement()` method on Application? - not necessary for this PR)